src/main/webapp/js/user-page-loader.js

```diff
@@ -101,16 +101,24 @@ function fetchAboutMe(){
     if(aboutMe == ''){
       aboutMe = 'This user has not entered any information yet.';
     }
-    
+
     aboutMeContainer.innerHTML = aboutMe;
 
   });
 }
 
+function createMDE() {
```

Comment:
Again, let's try to avoid duplication. Maybe extract it as a file somewhere

Reply:
Thanks for the comments! I will fix it now.